docker/docker-compose.1804.51.yaml

```diff
@@ -0,0 +1,19 @@
+version: "3"
+
+services:
+
+  runtime-setup:
+    image: swift-nio-ssl:18.04-5.1
```

Comment:
same comment here as in http2. We should have one of them on 16.04 and the other one on 18.04